extension/background/intentRunner.js

```diff
@@ -17,10 +18,15 @@ const METADATA_ATTRIBUTES = new Set([
   "example",
   "examples",
   "match",
+  "followup",
 ]);
 
 export const intents = {};
 let lastIntent;
+// This is a copy of lastIntent that gets nullified when the pop up
+// window closes.
+let lastIntentForFollowup;
+let globalListenForFollowup = settings.getSettings().listenForFollowup;
```

Comment:
I put this in the preferences view. Is that not the right place? I put it there because 1) I wasn't sure if you'd want continuous followups and 2) I thought the user may get to decide

Here's a screenshot for reference
![Screen Shot 2020-04-01 at 6 41 10 PM](https://user-images.githubusercontent.com/24993360/78193406-735a8180-7448-11ea-97af-b110f4472685.png)